node-api/engines/echo/validator.go

```diff
@@ -119,10 +119,8 @@ func ValidateUint64Dec(value string) bool {
 	if value == "" {
 		return true
 	}
-	if _, err := strconv.ParseUint(value, 10, 64); err == nil {
-		return true
-	}
-	return false
+	_, err := math.U64FromString(value)
```

Comment:
`math.U64FromString` is based on `strconv.ParseUint` which returns ErrSyntax in case of empty string.
We could avoid the `value == ""` test above, but I would avoid pulling up third libraries errors and just explicitly do the check ourselves